src/include/pebblesdb/filter_policy.h

```diff
@@ -47,6 +47,10 @@ class FilterPolicy {
   // This method may return true or false if the key was not on the
   // list, but it should aim to return false with a high probability.
   virtual bool KeyMayMatch(const Slice& key, const Slice& filter) const = 0;
+
+  // accounting the in-memory usage for the filter.
+  mutable size_t byte_size;
+  mutable unsigned long filter_count;
```

Comment:
I am not seeing where this filter_count is incremented. Was it not added into the PR by mistake?

Reply:
Sorry, I forgot to add this to the staging but I've made this change now